go/client/team_helper.go

```diff
@@ -51,3 +51,11 @@ func ParseUserAndRole(ctx *cli.Context) (string, keybase1.TeamRole, error) {
 	}
 	return username, role, nil
 }
+
```

Comment:
should this be a general `parseBoolean(ctx *cli.Context, var_name string, default_value boolean)`? unsure because other methods here aren't generic